### PR TITLE
Add Hint Card

### PR DESCRIPTION
**Pull request by ckopecky**

### CODE REVIEW

**Comment by ckopecky** [comment]
Screen Shot 2019-11-09 at 7 51 31 PM

Screen Shot 2019-11-09 at 7 53 08 PM

Screen Shot 2019-11-09 at 7 54 49 PM

File: src/styles/Card.js [hunk 1]
```diff
@@ -62,4 +61,30 @@ const CardPadding = styled.div`
...
+      color: ${colors.lighterGrey}
+    }
+  }
+`;
+
+const HintStyle = styled(Card)`
+
+  color: ${colors.lightGrey};
+  font-weight: ${typography.light};
+  border: 5px solid ${colors.blue};
```
Comment (bdougie) [inline src/styles/Card.js]: update: `  border: ${borderRadius} solid ${colors.blue};
`

File: src/styles/DoneChecking.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+import {borderRadius} from "./variables";
+
+const DoneChecking = styled.img`
+  width: 30%;
+  -moz-border-radius: 5px;
+  -webkit-border-radius: ${borderRadius};
+  border-radius: 5px;
```
Comment (bdougie) [inline src/styles/DoneChecking.js]: use `borderRadius` instead of `5px`?

File: stories/2-Card.stories.js [hunk 3]
```diff
@@ -130,3 +133,30 @@ export const NoteCard = () => (
...
+    <HintStyle style={{minWidth: "15%", width: "15%"}}>
+      <div id="context-div" style={{padding: "0 0 20px"}}>
+        <h1>Hint</h1>
+        <p>abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz</p>
+      </div>
+      <DoneChecking style={{width: "80%"}} src={done_checking}/>
+    </HintStyle>
+    <HintStyle style={{border: "none", minWidth: "15%", width: "15%"}}>
+      <PrimaryWithText style={{minWidth: "15%", width: "15%"}}/>
+    </HintStyle>
```
Comment (bdougie) [inline stories/2-Card.stories.js]: Why the inline styles?

File: stories/1-Button.stories.js [hunk 4]
```diff
@@ -13,3 +13,11 @@ export const primary = () => (
 );
 
 export const secondary = () => <Button onClick={action("clicked")}>Cancel</Button>;
+
+export const PrimaryWithText = () => (
+  <div>
+    <h2 style={{fontFamily: "Roboto", color:"grey", fontSize: "1.0rem"}}>Set goals directly on Github</h2>
+    <Button style={{width: "90%"}} primary>View Your Data</Button>
```
Comment (bdougie) [inline stories/1-Button.stories.js]: The preference here would be re-usability. Can the inline style be moved to a `<ButtonCardStyle />`?

**Comment by bdougie** [comment]
Let a few minor comments. Everything else looks great.

File: stories/2-Card.stories.js (hunk 3 above)
Comment (bdougie) [reply stories/2-Card.stories.js]: actually disregard. It is not worth addressing here, I can address myself in a future PR when I implement this.

File: stories/1-Button.stories.js (hunk 4 above)
Comment (bdougie) [reply stories/1-Button.stories.js]: actually disregard. It is not worth addressing here. 

I can address in my future implementation PR.

File: src/styles/Card.js (hunk 1 above)
Comment (ckopecky) [reply src/styles/Card.js]: ✔️

File: src/styles/DoneChecking.js (hunk 2 above)
Comment (ckopecky) [reply src/styles/DoneChecking.js]: ✔️

File: stories/1-Button.stories.js (hunk 4 above)
Comment (ckopecky) [reply stories/1-Button.stories.js]: I actually thought about that today. I can probably address that if you'd like me to 👍  Just let me know

**Comment by ckopecky** [comment]
@bdougie ready.

**Pull request merged**